exporter/opentelemetry-exporter-opencensus/src/opentelemetry/exporter/opencensus/metrics_exporter/__init__.py

```diff
@@ -34,6 +36,14 @@
 
 DEFAULT_ENDPOINT = "localhost:55678"
 
+# In priority order. See collector impl https://bit.ly/2DvJW6y
+OT_LABEL_PRESENCE_TO_RESOURCE_TYPE = [
```

Comment:
Unless this has to be imported elsewhere it would be better to have it in the method that uses it.

Reply:
Its not imported elsewhere, but it is constant and doesn't need to be re-instantiated. Is it alright with you if I just add a private `_` prefix?